packages/client/types/constEnums.ts

```diff
@@ -331,6 +331,7 @@ export const enum TierLabel {
 
 export const enum Threshold {
   MAX_NUMBER_OF_TASKS_TO_LOAD = 1000,
+  OAUTH_REFRESH = 1209600000, // 14 days
```

Comment:
if the cronjob runs every ~14 days & we have a var here for every 14 days does that mean the 2 are tightly coupled? 
not sure

Reply:
let's remove this here & make it a mutation variable.
then, then cronjob can tell us the threshold like `mutation { oauthRefresh(before: 2020-05-01) }`
that way we can call this ad-hoc 🎉

Reply:
That's a very good idea, done!